### PR TITLE
Issue 1657 - Update the deployment doc with the cluster deployment

### PR DESCRIPTION
**Pull request by linggao**

### CODE REVIEW

File: doc/deployment_string.md [hunk 1]
```diff
@@ -30,33 +36,41 @@ Because Horizon uses the Docker API to start the containers, many of the fields
     - `command`: `["--myfirstarg","argvalue",...]` - override the start CMD specified the dockerfile, or append to the ENTRYPOINT specified in the dockerfile.
     - `network`: `"host"` - start the container with host network mode. When network is set to host, the service can only be deployed to nodes with property openhorizon.allowPrivileged set to true.
 
+## clusterDeployment String Fields
+
+Because Horizon uses operator to deploy the applications in a Kubernetes cluster, the `clusterDeployment` containes the contents of the operator yaml archive files.
```
Comment (MaxMcAdam) [inline doc/deployment_string.md:41]: typo: "containes"

File: doc/deployment_string.md [hunk 2]
```diff
@@ -1,20 +1,26 @@
+
 # Horizon Deployment Strings
 
-When defining services in the Horizon Exchange, one of the fields that needs to be specified is `deployment`, which tells Horizon how the docker images should be deployed on the edge node. This page documents what can be specified in that string.
+When defining services in the Horizon Exchange, the deployment field defines how the service will be deoployed. There are two deployment fields for a service, `deployment` and `clusterDeployment`. You can define either one or both depending on the type of the service you want it to be. The `deoplyment` field is used by an edge node which tells Horizon how the docker images should be deployed on the node. The `clusterDeployment` field is used by an Horizon agent running in a Kubernetes cluster. It tells Horizon how the application should be deployed in the Kubernetes cluster.
```
Comment (MaxMcAdam) [inline doc/deployment_string.md:4]: typos: "deoployed" and "deoplyment"

Comment (linggao) [reply doc/deployment_string.md:4]: fixed

File: doc/deployment_string.md (hunk 1 above)
Comment (linggao) [reply doc/deployment_string.md:41]: fixed

**Pull request merged**